Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -268,14 +268,20 @@ public void ResolveAbsolutePositionedElement ()
     [Test]
     public void ResolveBorderElementA ()
     {
-      var home = Start();
+      // TODO: Remove when RM-7187 is resolved, use the RetryAttribute provided by NUnit instead.
```

Comment:
```suggestion
```

Reply:
We don't do a backport of NUnit 3, so there is no need to add the todo comments in the backport